src/routes/safe/components/Transactions/TxList/BatchExecute.tsx

```diff
@@ -191,6 +194,10 @@ export const BatchExecute = React.memo((): ReactElement => {
     toggleModal()
   }
 
+  if (!account) {
+    return null
+  }
```

Comment:
I think we should give some feedback to the user instead of simply not displaying anything here. I believe the fail text should handle this for us.